packages/cosmic-swingset/lib/kernel-stats.js

```diff
@@ -1,11 +1,147 @@
 import { MeterProvider } from '@opentelemetry/metrics';
 import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
 
+import makeStore from '@agoric/store';
+
 import {
   KERNEL_STATS_SUM_METRICS,
   KERNEL_STATS_UPDOWN_METRICS,
 } from '@agoric/swingset-vat/src/kernel/metrics';
 
+export const DEFAULT_METER_PROVIDER = new MeterProvider();
+
+export const HISTOGRAM_SECONDS_LATENCY_BOUNDARIES = [
```

Comment:
For the record (no pun intended! ok maybe a little intended) I'll mention that I still find it really weird that we have to pre-bin the values we record and send to Prometheus. I really wish we had a service that just let us send raw how-long-did-X-take numbers and let the backend figure out the statistics. I expect the graphs we get out of this to be difficult to read without a lot of mental prep work.

Reply:
Yeah.  This is one of the reasons I chose to use OpenTelemetry and not Prometheus directly.  We will be able to gain this feature when OTEL figures out how to implement it (and it is already on their TODO list), and possibly switch to a different metrics backend.